src/main/java/org/kohsuke/github/GHCommentAuthorAssociation.java

```diff
@@ -26,7 +26,8 @@ public enum GHCommentAuthorAssociation {
     /**
      * Author is a placeholder for an unclaimed user.
      */
-    MANNEQUIN,    /**
+    MANNEQUIN,
```

Comment:
Hello @bitwiseman,

I wanted to express my gratitude for your review of my PR and for the adjustment you made to the 'MANNEQUIN' field. It was a detail I overlooked, and your intervention not only corrected it but also improved the overall quality of the code. Your guidance and attention to detail are greatly appreciated.

This experience has been a valuable lesson for me in terms of attention to detail and has motivated me to be more thorough in my future contributions.

Thank you again for your support and for maintaining such a high standard in the project.